controller/src/main/java/io/pravega/controller/store/stream/ScaleMetadata.java

```diff
@@ -12,23 +12,27 @@
 import lombok.Data;
 
 import java.util.List;
+import lombok.AllArgsConstructor;
+import lombok.NoArgsConstructor;
 
 @Data
+@NoArgsConstructor
```

Comment:
We don't want a no args constructor.

Reply:
This constructor  is needed for JAX-RS deserialization.
I did not find how it was working without this constructor before. As far as I know JAX-RS does not use tricks for bean instantiation. Anyway they are dropping every way to construct objects bypassing the constructor in future releases of java. Maybe Jersey detects java10 and does not try to force the construction. I can debug more if you prefer

If this bean is a pojo for JAX-RS then it should have this constructor

Anyway a test about the REST API will fail due to an error on unmarshalling this kind of bean because of the lack of the constructor